taplo.toml

```
@@ -0,0 +1,31 @@
+[formatting]
+# Enable alignment of values in tables Align entries vertically
+align_entries = true
+
+# Align comments
+align_comments = true
+
+# Add trailing commas in arrays/tables
+array_trailing_comma = true
+table_trailing_comma = true
+
+# Indentation (spaces)
+indent_string = "  "
```

Can we use 4 spaces for indents please. That's what we've used in the code so that will make it consistent.
```suggestion
indent_string = "    "
```